spec/locales/en/translation.json

```diff
@@ -50,6 +50,7 @@
     "load": "Load",
     "logo": "Logo",
     "metadataTooltip": "View information/metadata about this object",
+    "mirrorTooltip": "Mirror image",
```

Comment:
Yeah totally! I asked that in the Slack channel and didn't get a response. Here I'm just following the pattern that is existing.